app/pages/index.tsx

```diff
@@ -61,12 +61,6 @@ const Home: NextPage = () => {
     for (const sensorId in responseJson.brighterBins) {
       sensorsArray.push(responseJson.brighterBins[sensorId]);
     }
-    for (const sensorId in responseJson.realFakeSensors) {
-      sensorsArray.push(responseJson.realFakeSensors[sensorId]);
-    }
-    for (const sensorId in responseJson.sensationalSensors) {
-      sensorsArray.push(responseJson.sensationalSensors[sensorId]);
-    }
```

Comment:
right this is referencing the old react code which is not in use! should we add more red here and delete the react app too? @mikevespi

Reply:
yeah I think it would make sense to just the whole /app folder if possible